test/Calculations/test_FirstQuartile.py

```diff
@@ -0,0 +1,33 @@
+from src.Calculations.FirstQuartile import FirstQuartile, get_calculation
+from pytest import approx
+
+
+def test_get_calculation():
+    """
+    Make sure the get_calculation function returns the calculation
+    """
+    assert get_calculation() == FirstQuartile
+
+
+def test_first_quartile():
+    """
+    Test functionality of the Average class
```

Comment:
Please fix the comment here (and in all test cases) to refer to the calculation being tested